README.md

```diff
@@ -43,6 +43,11 @@ Run the `run.py` script to perform training / classification. The first time `ru
 
 1. Clone https://hg.mozilla.org/mozilla-central/.
 2. Run `./mach vcs-setup` in the directory where you have cloned mozilla-central.
+3. Enable the pushlog, hgmo and mozext extensions. For example, if you are on Linux, add the following to the `~/.hgrc` file:
+    - pushlog = ~/.mozbuild/version-control-tools/hgext/pushlog
```

Comment:
Sorry, one last comment. Somebody could be confused and copy the whole thing from here, so don't use - but wrap everything in ```.